### PR TITLE
feat: rm xargs; use shorter one-liner

### PR DESCRIPTION
**Pull request by chadoh**

### CODE REVIEW

File: src/counter/updating.md [hunk 1]
```diff
@@ -42,10 +44,10 @@ impl Default for Counter {
 If you're following along with your own code, you can build and re-deploy to see the new default:
 
 ```bash
-raen build --release -q | xargs -I {} near dev-deploy --wasmFile {}
+near dev-deploy $(raen build --release -q)
 ```
 
-(This [uses xargs to](https://stackoverflow.com/a/42631216/249801) pipe the output of `raen build` to the `wasmFile` argument of `near dev-deploy`.)
+(This [uses `bash` interpolation](https://stackoverflow.com/a/23656236/249801) put the output of `raen build` in the correct spot in the `near dev-deploy` command. You can try `raen build --release` and `raen build --release -q` on their own to understand the compound command.)
```
Comment (willemneal) [inline src/counter/updating.md]: This makes me want to make our own dev-deploy soooo badly.  I think it's worth it before NEARCON so I'll add it to our list. This is only valid if using a bash shell so wrapping it in the CLI binary would be the most portable.

Comment (willemneal) [reply src/counter/updating.md]: For now I think a note that this only works on bash should be good enough for now.

Comment (chadoh) [reply src/counter/updating.md]: Do you think it's enough that I say "This uses `bash` interpolation"? Is that a good way to say it?

Comment (willemneal) [reply src/counter/updating.md]: It's a command substitution: https://hyperpolyglot.org/unix-shells#cmd-subst-note

but yeah I think that should be enough.

Comment (chadoh) [reply src/counter/updating.md]: I updated the link to point to yours, and changed the language to say "command substitution." I also made the "bash-only" thing more explicit.

**Pull request merged**